apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -143,11 +144,27 @@ router.get('/*', (req, res, next) => {
       }
 
       if ('jobSequence' in fileEdit) {
-        fileEdit.jobSequence.jobs.forEach((item) => {
-          if (item.type === 'git_push' && item.status === 'Error') {
+        // New case: single job for the entire operation. We check the flag
+        // we would have set when the job executed to determine if the push
+        // succeeded or not.
+        if (!fileEdit.jobSequence.legacy) {
+          const job = fileEdit.jobSequence.jobs[0];
+
+          // We check for presence of the `pushed` key to determine if we
+          // attempted a push, and we check for the value to know if the push
+          // succeeded or not.
```

Comment:
We could also distinguish between these with separate `pushAttempted` and `pushed` flags? 🤷

Reply:
I think it would be marginally clearer to use two flags (`pushAttempted` and `pushSucceeded`) rather than the convention of using the presence of a key but with a `false` value to indicate that the attempt started.

Reply:
I neglected to do this on this PR before merging 😢 I made this change over on https://github.com/PrairieLearn/PrairieLearn/pull/8005 in f1d1a91